pkg/importer/importer_test.go

```diff
@@ -103,6 +104,42 @@ func TestLoadXSDFromTestFiles(t *testing.T) {
 	})
 }
 
+func generateSyslFromSpannerSQL() (string, error) {
+	absFilePath, err := filepath.Abs(`../../tests/test-spanner.sql`)
+	if err != nil {
+		return "", err
+	}
+	logger, _ := test.NewNullLogger()
+	imp, err := Factory(absFilePath, nil, logger)
+	if err != nil {
+		return "", err
+	}
+	imp.WithAppName("customeraccounts").WithPackage("retail")
+	return imp.Load(absFilePath)
+}
+
+func readGoldenSyslForSpannerSQL() (string, error) {
+	absFilePath, err := filepath.Abs(`../../tests/accounts.golden.sysl`)
+	if err != nil {
+		return "", err
+	}
+	bytes, err := ioutil.ReadFile(absFilePath)
+	return string(bytes), err
+}
+
+func TestCompareSpannerSQLWithGolden(t *testing.T) {
+	// Load generated sysl module
+	genSysl, err := generateSyslFromSpannerSQL()
+	require.Nil(t, err)
```

Comment:
Nit: `require.Nil` is odd. AFAICT we're using `testify` elsewhere, so you probably want `assert.NoError(t, err)`.

Reply:
Not sure if there is a difference, a lot of the code still uses require.Nil. anyways a small change.

Reply:
Sorry yeah, I just looked it up and require is probably right in this instance. Both require and assert are part of the same library, but `require` terminates immediately if it fails, whereas `assert` can allow execution to continue to collect more errors.

`require` is probably right in this instance.